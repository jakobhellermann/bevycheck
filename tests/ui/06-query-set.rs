#![allow(unused)]
use bevy::prelude::*;

struct Foo;
struct Bar;

#[bevycheck::system]
fn system(
    _: QuerySet<(
        Query<Transform, With<Foo>>,
        Query<&mut Transform, Without<Foo>>,
    )>,
) {
}

fn main() {
    IntoSystem::into_system(system);
}
