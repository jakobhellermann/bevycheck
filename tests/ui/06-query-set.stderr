error: invalid query parameter
  --> $DIR/06-query-set.rs:10:15
   |
10 |         Query<Transform, With<Foo>>,
   |               ^^^^^^^^^
   |
   = note: `Transform` is not a valid query type
   = help: if you want to query for a component, use `&Transform` or `&mut Transform`
